### PR TITLE
Cleans up details a bit. More compact, order more relevant. Fixes #35

### LINKED ISSUE #35: Improve shelter popup details - more compact, better flow
## Issue Overview

That describes it all really

### PR DESCRIPTION
**Pull request by plocket**
## Description

Fixes #35

Shelter details should now look like this:
- address
- directions
- phone icon, phone number
- 'Pets' label, pets details
- Notes label
- Notes details

## List of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] This pull request relates to a ticket.
- [x] This code is tested.
- [x] I want my code added to the response theme.

### CODE REVIEW

**Comment by plocket** [comment]
@miklb : What more does this need before it can be merged?

**Comment by plocket** [comment]
@omnilord : I can't add reviewers, so I'll just ping you here for a review.

**Comment by plocket** [comment]
@omnilord : done as far as I can tell

**Comment by omnilord** [comment]
Looks decent.  Removing the directions arrow may be a bit confusing since it is a ubiquitous visual cue for "get me directions to this location." Thoughts on making it more stylistically a better fit kind of like with the phone icon?

**Comment by plocket** [comment]
The arrow doesn't help during the day because it's white. I can change the color. Or show me the phone icon and I can try. Does this theme have fontawesome?

**Comment by plocket** [comment]
@omnilord

**Comment by miklb** [comment]
No, it uses SVGs. I have an undocumented gulp task to inline svgs Previously they were just hard coded.

**Comment by miklb** [comment]
I might be thinking of another project though, I’ve done similar.

**Comment by plocket** [comment]
Oh, shoot, I was thinking of a different PR from earlier. I can restore this one.

**Comment by plocket** [comment]
Added back in. I also noticed it was a little hard to tell that the links were links. It's hard to see the blue color. At least, it is for me. I added underlines. Let me know if that's a problem.

**Comment by miklb** [comment]
looks good to me. Probably want a new issue for the `null` on notes unless someone has a quick solution to add to this PR.

File: assets/css/_inc/map.css [hunk 1]
```diff
@@ -177,6 +182,10 @@
     line-height: .7em;
 }
 
+#icon-directions {
```
Comment (omnilord) [inline assets/css/_inc/map.css]: Did you mean to leave this style empty?

Comment (plocket) [reply assets/css/_inc/map.css]: I did not! Thanks for catching that!

**Comment by plocket** [comment]
@miklb : If you think it's related enough to this PR, I can take a stab. Let me know if it'd just be better to merge what we have so other merges can move forward.

**Comment by miklb** [comment]
Not a blocker on my end. If you wouldn't mind opening the issue that would be helpful to remember. I'm working through some organizing now.

**Comment by plocket** [comment]
@miklb : It looks like it was maybe taken care of? I'm getting 'Unknown' for a lot of these and not seeing any `null`s.

@miklb , @omnilord : Is this ready to merge?

**Comment by plocket** [comment]
@miklb : Nvm, just found one I can test with!

**Comment by plocket** [comment]
@miklb : I've figured out a way. I'll make an issue and then a PR once this other stuff has been merged.

File: assets/css/main.css [hunk 2]
```diff
@@ -436,14 +436,21 @@ a:active {
...
+    text-decoration: underline;
 }
 .mapboxgl-popup-content svg {
     height: 1em;
 }
 .shelter-popup .mapboxgl-popup-close-button {
     font-size: 2.5em;
     line-height: .7em;
+}
+#icon-directions {
```
Comment (omnilord) [inline assets/css/main.css]: Can I get you to do a `gulp build` then push the updated `main.css` and `main.css.map`?

**Comment by plocket** [comment]
@omnilord : Done. Didn't know it didn't happen before and not sure why... a puzzle for another time.

**Pull request merged**